tests/mir-opt/inline/issue_76997_inline_scopes_parenting.rs

```diff
@@ -1,8 +1,12 @@
-// skip-filecheck
 // Tests that MIR inliner can handle `SourceScopeData` parenting correctly. (#76997)
 
 // EMIT_MIR issue_76997_inline_scopes_parenting.main.Inline.after.mir
 fn main() {
+    // CHECK-LABEL: fn main(
+    // CHECK: scope 2
+    // CHECK-NEXT: debug x
+    // CHECK-NEXT: scope 3
+    // CHECK-NEXT: debug y
```

Comment:
```suggestion
```suggestion
    // CHECK: scope 1 {
    // CHECK-NEXT: debug f
    // CHECK-NEXT: scope 2 (inlined main::{closure#0}) {
    // CHECK-NEXT: debug x
    // CHECK-NEXT: scope 3 {
    // CHECK-NEXT: debug y
    // CHECK-NEXT: }
    // CHECK-NEXT: }
    // CHECK-NEXT: }
```

We want to verify that the scopes nest correctly. We need the braces to do that.